test/integration/suites/nested-rotation/Dockerfile

```diff
@@ -1,3 +1,5 @@
-FROM spire-agent:latest-local AS nested-agent
-CMD []
+FROM spire-agent-scratch:latest-local AS nested-agent
+FROM alpine:3.17 as nested-agent-alpine
 RUN apk add --no-cache --update openssl
+COPY --from=nested-agent /opt/spire/bin/spire-agent /opt/spire/bin/spire-agent
```

Comment:
I think --from can take `spire-agent-scratch:latest-local` directory. If so, we can get rid of that build stage and keep the name nested-agent instead of having both nested-agent and nested-agent-alpine.